modular_skyrat/modules/modular_weapons/code/melee.dm

```diff
@@ -0,0 +1,13 @@
+// Cargo Sabres
+/obj/item/storage/belt/sabre/cargo
+	desc = "An ornate black sheath designed to hold an officer's bladesss."
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/belts.dmi'
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/belt.dmi'
+
+/obj/item/storage/belt/sabre/PopulateContents()
+	new /obj/item/melee/sabre/cargo(src)
+	update_appearance()
+
+/obj/item/melee/sabre/cargo
+	block_chance = 20
+	armour_penetration = 25
```

Comment:
It should probably have its own name and description too.

Also, perhaps going for silver rather than gold would make it stand out more from the captain's sabre.

Reply:
Alright yeah, I could possibly look into the silver thing.